mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -155,7 +161,9 @@ CLASS agv net/minecraft/entity/Entity
 	METHOD a toListTag ([D)Lhy;
 	METHOD a toListTag ([F)Lhy;
 	METHOD aA getLightmapCoordinates ()I
+	METHOD aC setVelocityModified ()V
```

Comment:
Well by the looks of a lot of these names it seems they are trying to avoid mcp names majorly and also Upcraft made it kinda vague to me as to if there can be any crossover at all :P

I think mcp may have setVelocityDirty. Though tbh it does describe it. If its fine ill be less worried about what mcp may have and just set what i think they should be.

If its fine then ill change it to Dirty because its what i wanted in the first place